src/CoreShop/Bundle/PaymentBundle/Resources/config/doctrine/model/Payment.orm.xml

```diff
@@ -12,6 +12,7 @@
         <field name="state" column="state"/>
         <field name="orderId" column="orderId" type="integer"/>
         <field name="datePayment" column="datePayment" type="datetime"/>
+        <field name="details" column="details" type="json_array" />
```

Comment:
Please be aware that this type is [deprecated since DBAL 2.6](https://www.doctrine-project.org/projects/doctrine-dbal/en/2.13/reference/types.html#json-array) and was removed in DBAL 3 (`json` should be used instead).

As far as I can see, this is use first usage of the `json_array` type in CoreShop.
If so, it's probably better to directly use `json` instead.
If not, maybe all other usages could be converted before the final release of CoreShop 3, so it's possible to use it together with DBAL 3 (here's a blog post about [upgrading `json_array` to `json`](https://dunglas.fr/2022/01/json-columns-and-doctrine-dbal-3-upgrade/)).

Reply:
thanks, changed